_episodes/02-practice-learning.md

```diff
@@ -173,19 +172,18 @@ perhaps illustrated as:
 
 > ## Mapping a Mental Model
 >
-> 1) On a piece of paper, draw a simplified concept map of the same concept you discussed in the last activity, but this time without the analogy. 
+> 1. On a piece of paper, draw a simplified concept map of the same concept you discussed in the last activity, but this time without the analogy. 
 > What are 3-4 core 
-> concepts involved? How are those concepts related? (Note: if you would like to try out an online tool for this exercise, visit <https://excalidraw.com> .)
->  
-> 2) In the Etherpad, write some notes on this process. Was it difficult? Do you think it would be a useful exercise prior to teaching about your topic?
+> concepts involved? How are those concepts related? (Note: if you would like to try out an online tool for this exercise, visit <https://excalidraw.com>.)
+> 2. In the Etherpad, write some notes on this process. Was it difficult? Do you think it would be a useful exercise prior to teaching about your topic?
 > What challenges might a novice face in creating a concept map of this kind?  
+> 
 > This exercise should take about 5 minutes.
 {: .challenge}
 
 ## Misconceptions
 
-The mental model above connects a ball to the water it can displace, recognizing that 'more' ball can move 'more' water. This mental model is perfectly functional for a child who wants to have fun splashing water around.  
-It may endure in this way for several years of beaches and bathtubs.
+The mental model above connects a ball to the water it can displace, recognizing that 'more' ball can move 'more' water. This mental model is perfectly functional for a child who wants to have fun splashing water around. It may endure in this way for several years of beaches and bathtubs.
```

Comment:
I know we're not very consistent at this overall, but it's actually better for version control to keep that newline so that sentences are separate lines, it makes a diff easier to read.

Reply:
If that sentence is part of the previous paragraph - we should probably remove the new line. The sentence before it also does not start on the new line so I do not see why this sentence is special.

Reply:
I know that we're not consistent, my intentions was that the better choice, for version control is to add an additional new line in the markdown.  Newlines are not paragraph breaks, blank lines are paragraph breaks. I work in more strict environments a lot though, so here I'm okay to defer to @karenword

Reply:
I am attaching the screenshot here to show what the problem is - I think we have 2 blank spaces which causes the sentence to start on a new line, rather than continue in the same paragraph. I agree that in source Markdown having sentences start on a new line is handy for easier diffs - but this is a separate issue I believe. Took me a while to figure out what is wrong and that @brownsarahm and I were talking about slightly different issues.
<img width="1155" alt="Screenshot 2022-03-04 at 16 34 01" src="https://user-images.githubusercontent.com/631719/156803382-b955f229-e6b1-40fc-93fd-9289b1deb0eb.png">

Reply:
I've done the easy fix in PR #1327.

Reply:
ohhh sorry, i had not rendered, i was looking at the source and white spaces at the end of the line also make a new paragraph.

Reply:
thank you for your patience and diligence!